bindings/ruby/Cargo.toml

```diff
@@ -18,7 +18,7 @@
 [package]
 name = "opendal-ruby"
 publish = false
-version = "0.1.11"
+version = "0.0.0" # refers version at `lib/opendal_ruby/version.rb`
```

Comment:
We need this because we will use this field to generate the correct source tarball for Ruby. Is it possible to sync the version to `lib/opendal_ruby/version.rb` instead?

Reply:
Ok. I removed the `version` file. And I set the version by reading `Cargo.toml` in `opendal.gemspec`.

Users can't read `OpenDAL::VERSION` in their console (ruby REPL, `irb`). But most Ruby users use bundlers to manage dependencies in the real world. I don't see it as a problem.